srv/modules/runners/osd.py

```diff
@@ -157,6 +199,10 @@ def _call(self):
             log.error("No host found")
             return False
 
+        if not self.force:
+            print("Checking if OSD can be destroyed")
+            if self._wait_until_empty() is False:
```

Comment:
Minor nit  .. but maybe just `return self._wait_until_empty()` ?

```suggestion
            return self._wait_until_empty()
```

Reply:
It has to `pass` on True so that the remaining steps complete.

Reply:
ah yep, I missed that